cli/cmd/plugin/unmanaged-cluster/cmd/create.go

```diff
@@ -63,7 +64,8 @@ func init() {
 	CreateCmd.Flags().StringSliceVarP(&co.portMapping, "port-map", "p", []string{}, "Ports to map between container node and the host (format: '80:80/tcp' or just '80')")
 	CreateCmd.Flags().Bool("tty-disable", false, "Disable log stylization and emojis")
 	CreateCmd.Flags().BoolVar(&co.skipPreflightChecks, "skip-preflight", false, "Skip the preflight checks; default is false")
-	CreateCmd.Flags().StringVarP(&co.numNodes, "nodes", "n", "", "The number of nodes to deploy; default is 1 control-plane")
+	CreateCmd.Flags().StringVarP(&co.numContPlanes, "control-plane-node-count", "", "", "The number of control plane nodes to deploy; default is 1")
```

Comment:
Should these be `StringVar` instead of `StringVarP`? Or still determining what to use for the one-letter shorthand?

Reply:
Good catch - these probably don't need shorthands and can just be `StringVar`